rmgpy/data/kinetics/family.py

```diff
@@ -2130,7 +2152,7 @@ def getTrainingDepository(self):
             if depository.label.endswith('training'):
                 return depository
         else:
-            raise Exception('Could not find training depository in family {0}.'.format(self.label))
+            raise AttributeError('Could not find training depository in family {0}.'.format(self.label))
```

Comment:
Would a `DatabaseError` be more appropriate? I feel like `AttributeError` has a fairly specific scope.

Reply:
Sounds great! I'll make this change. :)